src/server/mappings/index.js

```diff
@@ -472,7 +472,9 @@ function getVisibleResults() {
 
 
 function getDefaultSortFields() {
-  return mappings.filter((mapping) => mapping.key === defaultSortFieldKeys);
```

Comment:
Der Vergleich `mapping.key === defaultSortFieldKeys` wird immer `false` ergeben, da `key` ein String und `defaultSortFieldKeys` ein Array ist.

Zudem müssen die Sortierfelder von `defaultSortFieldKeys` aus bestimmt werden, da sonst die Reihenfolge der Feld-Keys in `defaultSortFieldKeys` nicht berücksichtigt wird.

## Beispiel
Ausgehend von:
```js
const defaultSortFieldKeys = ['sorting_info_year', 'sorting_info_position'];
```

```js
const mappings = [
  {
    display_value: 'sortingInfo.position',
    showAsFilter: false,
    showAsResult: false,
    sortable: true,
    filter_types: [],
    key: 'sorting_info_position',
    value: 'sortingInfo.position',
  },
  {
    display_value: 'sortingInfo.year',
    showAsFilter: false,
    showAsResult: false,
    sortable: true,
    filter_types: [],
    key: 'sorting_info_year',
    value: 'sortingInfo.year',
  },
];
```

Da im vorherigem Code `mappings` durchgegangen wurde, ist er durch den `filter`-Vergleich zuerst auf `'sorting_info_position` gestoßen, da dieser in `mappings` vor `sorting_info_year` liegt. Erst danach wird `sorting_info_year` gefiltert.

Wir hätten dadurch eine verdrehte Sortierreihenfolge, da zuerst nach Position und im zweiten Schritt nach Jahr sortiert wird.
Geht man alternativ von `defaultSortFieldKeys` aus, wird die Reihenfolge beibehalten, weil zuerst das Mapping für `sorting_info_year` und danach erst das Mapping für `sorting_info_position` gesucht und gefunden wird.